cleanlab_studio/internal/tlm/validation.py

```diff
@@ -17,6 +17,36 @@
 )
 
 
+def validate_tlm_prompt_kwargs_constrain_outputs(
```

Comment:
[nit] this function could probably just be `validate_tlm_prompt_kwargs`? (in the case where we add more kwargs in the future)

Reply:
i purposely made this specific for constrain_outputs since as you can see that the validation logic for each kwargs is fairly complicated so better to have separate validation function for each of them.